default/project.hxp.tpl

```diff
@@ -0,0 +1,82 @@
+import lime.project.*;
+
+class Project extends HXProject
+{
+
+	public function new()
+	{
+		super();
+
+		meta = { packageName: ${PROJECT_NAME}, company: "HaxeFlixel", version: "0.0.1" };
+
+		app = { main: "Main", path: "Export", file: ${PROJECT_NAME}, preloader: "flixel.system.FlxPreloader", swfVersion:  11.8 };
+
+		defines.set("SWF_VERSION", "11.8");
+
+
+		sources.push("source");
+
+		// put haxe library names you want to use below
+		var libs = ["flixel", /*"flixel-addons",*/ /*"flixel-ui",*/ /*"nape"*/];
+		for (l in libs)
+			haxelibs.push(new Haxelib (l));
+
+		includeAssets("assets");
+
+		window.width = $ {WIDTH};

```

Comment:
There's a space between the `$` and `{` here, I doubt the replacement still works with that.